docs/user_guide/configuring.rst

```diff
@@ -12,27 +12,64 @@ you can configure in various ways. This page describes the options available to
 Configure project logo
 ======================
 
-To add a logo that's placed at the left of your nav bar, put a logo file under your
-doc path's _static folder, and use the following configuration:
+By default the theme will use the name of your project on the left side of the header navbar. This can be replaced by a Logo.
```

Comment:
```suggestion
By default the theme will use the value of `project` on the left side of the header navbar.
This can be replaced by a Logo image, and optionally a custom `html_title` as well.
```